### PR TITLE
add support for $SUDO_USER

### PR DESCRIPTION
**Pull request by sonnyp**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -3,9 +3,9 @@ const childProcess = require('child_process');
 const execa = require('execa');
 const mem = require('mem');
 
-function getEnvVar() {
+function getEnvVar(options) {
```
Comment (sindresorhus) [inline index.js:6]: I don't see the point in an option, you can just always check `env.SUDO_USER` as it's only defined when sudo is used.

Comment (sonnyp) [inline index.js:6]: it's a breaking change and is not always desirable that's why it's disabled by default

Comment (sindresorhus) [inline index.js:6]: When would it not be desirable?

Comment (sonnyp) [inline index.js:6]: If you want `root` when your program is run with sudo.
AFAIK there is no 'standard' as to how a program should behave when run with sudo so it's really up to the program author.

**Pull request closed without merging**